### ISSUE TITLE
Crash in SaveRealInfos()

### ISSUE THREAD

**Issue by ChillerDragon** [ChillerDragon/DDNetPP#56]
```
#0 - src/game/server/db_sqlite3.cpp:118:InitWorker -
#1 - src/game/server/db_sqlite3.cpp:51:WorkerThread -
#2 - src/game/server/score/file_score.cpp:88:SaveScoreThread -
#3 - src/game/server/gamecontext.cpp:1571:OnClientEnter -
#4 - src/game/server/gamecontext.cpp:1571:OnClientEnter -
#5 - src/game/server/gamecontext.cpp:1239:OnTick -
#6 - src/game/server/gameworld.cpp:282:Tick -
#7 - src/game/server/score/file_score.cpp:88:SaveScoreThread -
#8 - src/game/server/score/file_score.cpp:88:SaveScoreThread -
#9 - src/game/server/score/file_score.cpp:88:SaveScoreThread -
#10 - src/game/server/score/file_score.cpp:88:SaveScoreThread -
#11 - src/game/server/gamecontext.cpp:5991:OnMessage -
#12 - src/game/server/entities/character.cpp:4852:SaveRealInfos -
```
after executing ``kill_pl all`` command the 3rd time.

**Comment by ChillerDragon** [comment]
@fokkonaut ur code :p

**Comment by fokkonaut** [comment]
the 3rd time xD

**Comment by ChillerDragon** [comment]
idk i tried to give some random info xd
so you know many tees were dieing and it doesnt happen every time so its not 100% sure that thats the real problem

**Comment by ChillerDragon** [comment]
Fixed by 3313e5dc64f9b2526d1e3fd5b740267d2e7e0eb1
